src/utils.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const path = require("path");
+
+const getDataFromFile = () => {
+  const jsonData = fs.readFileSync(
+    path.join(__dirname, "../db/db.json"),
+    "utf-8"
+  );
+
+  const data = JSON.parse(jsonData);
```

Comment:
No need to create and store a variable here just to simply return it. We can just directly return this statement:

`return JSON.parse(jsondata)`